src/cluster/purging.rst

```diff
@@ -129,45 +131,52 @@ consistency within the main database.
 
 Config Settings
 ====================================
-These settings ca be updated in the default.ini or local.ini:
+These settings can be updated in the default.ini or local.ini:
+
++-----------------------+--------------------------------------------+----------+
+| Field                 | Description                                | Default  |
++=======================+============================================+==========+
+| max_document_id_number| Allowed maximum number of documents in one | 100      |
+|                       | purge request                              |          |
++-----------------------+--------------------------------------------+----------+
+| max_revisions_number  | Allowed maximum number of accumulated      | 1000     |
+|                       | revisions in one purge request.            |          |
```

Comment:
This is a super anal nit-pick, but maybe remove the period, since none of the other descriptions have periods?
```suggestion
|                       | revisions in one purge request             |          |
```